cmds/deploy.go

```diff
@@ -160,7 +166,7 @@ func NewCmdDeploy(f *cmdutil.Factory) *cobra.Command {
 
 						tmpl.Parameters = append(tmpl.Parameters, tapi.Parameter{
 							Name:  "DOMAIN",
-							Value: domain,
+							Value: apidomain,
```

Comment:
Shouldn't this be added as a new template variable? Both domain & apidomain would be required i think.